services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -138,6 +138,14 @@ class DynamicSidecarSettings(BaseCustomSettings):
             "platform this value is set to 1 hour."
         ),
     )
+    DYNAMIC_SIDECAR_API_RESTART_CONTAINERS_TIMEOUT: PositiveFloat = Field(
+        1 * MINS,
+        description=(
+            "When saving and restoring the state of a dynamic service, depending on the payload "
+            "some services take longer or shorter to save and restore. Across the "
+            "platform this value is set to 1 hour."
```

Comment:
this comment confuses me.
First in your PR description you wrote that you do not save the state between restarts. Which I wonder if it's a good thing or not (I hope the frontend tells the user he will lose his/her work???)
and then here you say the contrary, and also that the value is set to 1hour but here I see 1 minute??

Reply:
Sorry for this, I forgot to change there description.
When you restart the container the user will not loose their state. Whatever is present on the disk will still be back after the restart.

Reply:
What about the 1 minute vs 1 hour in the comment?

Reply:
It was wrong. Just copied and pasted the block from above. Ignore that I've updated it.